### PR TITLE
Package Graph Documentation 

### PR DESCRIPTION
**Pull request by annacanal**
#231 Package Graph Documentation finished

### CODE REVIEW

File: braph/graph/are_compatible.m [hunk 1]
```diff
@@ -1,4 +1,10 @@
 function bool = are_compatible(g, m)
+% ARECOMPATIBLE checks whether graph and measure are compatible
```
Comment (egolol) [inline braph/graph/are_compatible.m:2]: ARE_COMPATIBLE

File: braph/graph/binarize.m [hunk 2]
```diff
@@ -1,4 +1,9 @@
 function B = binarize(A, varargin)
+% BINARIZE binarizes a matrix
+%
+% B = BINARIZE(A) binarizes the matrix A with 1 for values different than 0, and 0 for the 0 values.
```
Comment (egolol) [inline braph/graph/binarize.m:4]: just as a note, varargin does nothing for this function

File: braph/graph/measures/Assortativity.m [hunk 3]
```diff
@@ -1,14 +1,56 @@
...
         function assortativity = calculate(m)
-            g = m.getGraph();
-            A = g.getA();
+            % CALCULATE calculates the assortativity value of a graph
+            %
+            % ASSORTATIVITY = CALCULATE(M) returns the value of the assortativity of a
+            % graph.
+            
+            g = m.getGraph(); % graph from measure class
+            A = g.getA(); % adjency matrix of the graph
```
Comment (egolol) [inline braph/graph/measures/Assortativity.m:53]: for inline comments its two free spaces then the comment. examle:
line of code;  % comment

File: braph/graph/measures/Clustering.m [hunk 4]
```diff
@@ -1,14 +1,68 @@
...
+            % Admissible RULE options are:
+            % DIRECTEDTRIANGLESRULE = 'cycle' (default) - calculates CLUSTERING of a node using the cycle rule for directed graphs.
+            %                    'all' - calculates CLUSTERING of a node
+            %                    using the all rule for directed graphs.
+            %                    'middleman' - calculates CLUSTERING of a
+            %                    node using the middleman rule for directed graphs.
+            %                    'in' - calculates CLUSTERING of a node
+            %                    using the in rule for directed graphs.
+            %                    'out' - calculates CLUSTERING of a node
+            %                    using the out rule for directed graphs.
```
Comment (egolol) [inline braph/graph/measures/Clustering.m:46]: I just noticed until this point, but the comments do not respect the margin give by matlab.

File: braph/graph/binarize.m (hunk 2 above)
Comment (annacanal) [reply braph/graph/binarize.m:4]: But if we remove it we need to change several functions.
should we make the changes now??

File: braph/graph/measures/Clustering.m (hunk 4 above)
Comment (annacanal) [reply braph/graph/measures/Clustering.m:46]: Do you mean when for the RULE comments?

File: braph/graph/are_compatible.m [hunk 5]
```diff
@@ -1,4 +1,10 @@
 function bool = are_compatible(g, m)
+% ARE_COMPATIBLE checks whether graph and measure are compatible
+%
+% B = ARECOMPATIBLE(G, M) checks whether the graph G and the measure M are compatible
```
Comment (giovannivolpe) [inline braph/graph/are_compatible.m:4]: ARE_COMPATIBLE

File: braph/graph/binarize.m (hunk 2 above)
Comment (egolol) [reply braph/graph/binarize.m:4]: it was more a note for me or giovanni. but as a curiosity, what class should we change?

**Pull request merged**